networking/multiple_networks/secondary_networks/configuring-multi-network-policy.adoc

```diff
@@ -14,6 +16,8 @@ As a cluster administrator, you can configure a multi-network policy for any of
 * Bond Container Network Interface (CNI) over SR-IOV
 * OVN-Kubernetes additional networks
 
+Other types of additional networks, such as IP Virtual Local Area Network (IPVLAN), are not supported.
```

Comment:
Let's omit this. Macvlan and ipvlan should be supported. I don't think this supportable list changes.